lunatrace/bsl/hasura/migrations/lunatrace/1664218140551_alter_table_public_manifest_dependency_edge_add_column_id/down.sql

```diff
@@ -0,0 +1,5 @@
+alter table "public"."manifest_dependency_edge" drop column "id" cascade
+alter table "public"."manifest_dependency_edge" drop column "id";
```

Comment:
I'm still confused why edges don't just have all the info about their child in them as we had before, it's a one-to-one and it's just a few rows. Now we are getting into complexity here where it turns out that simple join table we had before now has its own identity we want to point to. It's fine but it feels like a consequence of that decision that I still don't quite understand. 

But, you gotta do what you gotta do.